website/content/docs/configuration/identity-access-management/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: Permissions index
-description: |-
-  Boundary's permissions model
+description: >-
+  Learn about Boundary's RBAC, allow-only permissions model. Understand how permissions are configured using grant strings and roles.
```

Comment:
not sure if this is needed, just a thought 

```suggestion
  Learn about Boundary's RBAC (Role-based access control), allow-only permissions model. Understand how permissions are configured using grant strings and roles.
```

Reply:
Yeah, I think it's helpful info and it fits inside of 160 characters.